search.go

```diff
@@ -266,18 +267,40 @@ func (s *Search) FullSearch(q string, fuzzy bool) ([]*SearchHit, int, time.Durat
 	return hits, int(res.Total), res.Took, s.isStale, nil
 }
 
+type Nodes []*Node
+
+// String matches https://github.com/sahilm/fuzzy Source interface
+func (n Nodes) String(i int) string {
+	return n[i].Title()
+}
+
+// Len matches https://github.com/sahilm/fuzzy Source interface
+func (n Nodes) Len() int {
+	return len(n)
+}
+
 // FilterSearch performs a narrow restricted prefix search on the
 // node's visible attributes (the title) plus tags & keywords.
 //
 // Does not use search index, as it's not possible to narrow field
 // scope on a per query basis. This means we'd need to keep a second
 // index just for filter searches. The simplistic approach used her is
 // "good enough" to fullfill the requirements.
-func (s *Search) FilterSearch(q string, fuzzy bool) ([]*Node, int, time.Duration, error) {
+func (s *Search) FilterSearch(q string, isFuzzy bool) ([]*Node, int, time.Duration, error) {
```

Comment:
It's okay to change this, for symmetry please change the name of the arg in `FullSearch()`, too.